.github/ISSUE_TEMPLATE.md

```diff
@@ -8,7 +8,9 @@
 - [ ] Update to an existing algorithm
 - [ ] Error
 - [ ] Proposal to the Repository
-
+- [ ] Other
+** If Other Please Specify What you Did:**
```

Comment:
Instead of writing:
** If Other Please Specify What you Did:**

Write:
** Specify What the Issue is:**

Reply:
That makes more sense I could not figure out what to write

Reply:
Please make the changes.

Reply:
Please make the changes.